builder/imports/nodejs.py

```diff
@@ -44,12 +47,15 @@ def install(self, env):
         self.install_dir = os.path.join(env.deps_dir, self.name)
         sh.mkdir(self.install_dir)
 
-        if current_os() == 'windows':
-            self.install_nvm_choco(env)
+        if current_arch() == "x86":
+            self.install_node_build(env)
```

Comment:
trivial: maybe name this `install_node_via_unofficial_build()`

this is similar to the existing `install_node_via_nvm()` function

also just calling it "build" made me thing it was something we were compiling locally